content/docs/engine-dev/tools/tiaf/maintaining-tiaf/general-information.md

```diff
@@ -0,0 +1,114 @@
+
+```
+ly_add_googletest(
+    NAME AZ::MyExample.Tests
+    LABELS REQUIRES_tiaf;TIAF_shard_test
+)
+
+```
+
+When using the `TIAF_shard_test` label, each individual test will be interleaved across the available shards. This delivers the greatest performance boost but increases the brittleness of the sharded test target as resource race conditions for badly behaving tests are more likely to manifest. On the contrary, `TIAF_shard_fixture` interleaves each individual fixture across the available shards, delivering less performance than `TIAF_shard_test` but less brittleness. Always test your targets when enrolling them into test sharding optimization to ensure that they behave well when running as shards.
+
+{{< note >}}
+The test target must be enrolled into native TIAF in order to benefit from test sharding, else the test target will not be run by TIAF.
+{{< /note >}}
+
+# What are the command line options for the TIAF runtimes?
+
+The following table details the command line options common to both the native and Python runtimes:
+
+|Option|Flag|Default|Description|
+|:----|:----|:----|:----|
+|Configuration File|-config=<filename>|<tiaf binay build dir>.<tiaf binary build type>.json|Path to the configuration file for the TIAF runtime.|
+|Test Impact Data File|-testimpactdatafile=<filename>|None|Optional path to a test impact data file that will used instead of that specified in the config file.|
+|Previous Test Runs File|-previousrundatafile=<filename>|None|Optional path to a test impact data file that will used instead of that specified in the config file.|
+|Change List File|-changelist=<filename>|None|Path to the JSON of source file changes to perform test impact analysis on.|
+|Global Timeout|-gtimeout=<seconds>|No timeout|Global timeout value to terminate the entire test sequence should it be exceeded.|
+|Test Target Timeout|-ttimeout=<seconds>|No timeout|Timeout value to terminate individual test targets should it be exceeded.|
+|Sequence Type|-sequence=<none, seed, regular, tia, tianowrite, tiaorseed>|None|The type of test sequence to perform, where: <br /><br />`none` runs no tests and will report a all tests successful.<br /><br />`seed` removes any prior coverage data and runs all test targets with instrumentation to reseed the data from scratch<br /> `regular` runs all of the test targets without any instrumentation to generate coverage data (any prior coverage data is left intact).<br /><br />`tia` uses any prior coverage data to run the instrumented subset of selected tests (if no prior coverage data a regular run is performed instead).<br /><br />`tianowrite` uses any prior coverage data to run the uninstrumented subset of selected tests (if no prior coverage data a regular run is performed instead) but the coverage data is not updated with the subset of selected tests.<br /><br />`tiaorseed` uses any prior coverage data to run the instrumented subset of selected tests (if no prior coverage data a seed run is performed instead).|
```

Comment:
update: report a all -> report all

Reply:
nit: if no prior coverage -> if there is not prior coverage

Reply:
Fixed 👍